portafly/doc/ux_specs/global_components/modal.md

```diff
@@ -71,3 +71,17 @@
           * the spinner will be replaced by the submit button
           * the input fields will be enabled again (to allow user eventually copy paste their content)
           * an inline warning message will be displayed above the form buttons -->
+
+### _'Delete confirmation'_ modal
+* Modal's header includes the name of the current API (Product or Backend) the user is trying to delete
+* Enter API name > text input field [required]
```

Comment:
@cntlsn Probably I'm missing the whole picture here. I read:
* Modal's header includes the name of the current API (Product or Backend) the user is trying to delete
* Enter API name > text input field [required]

Why is the user required to specify the API name already shown in the header?

Reply:
It's a kind of safety measure used when performing a destructive, potentially catastrophic action.

Reply:
From an accessibility viewpoint, having the API name three times in the same modal is noisy (header, prompt to action and actual typing). The header can be something more generic like `Confirm API deletion`

Reply:
@porueesq agreed, changed!